entrypoint.sh

```diff
@@ -72,7 +72,13 @@ if [ -z "$ETCD_NODE"]; then
   ETCD_NODE=$IP:2379
 fi
 
-echo "[kontena-lb] booting $KONTENA_SERVICE_NAME. Using etcd: $ETCD_NODE"
+if [ -z $KONTENA_STACK_NAME ] || [ "$KONTENA_STACK_NAME" == "null" ]; then
```

Comment:
Strikes me as odd that some of these are quoted, and others are not. Not that it should really matter, the `KONTENA_STACK_NAME` is trusted.

```sh
KONTENA_STACK_NAME="asdf -o -n asdf"; if [ -z $KONTENA_STACK_NAME ]; then echo empty; fi
```
```
+ KONTENA_STACK_NAME='asdf -o -n asdf'
+ '[' -z asdf -o -n asdf ']'
+ echo empty
```